examples/tv-casting-app/APIs.md

```diff
@@ -1,123 +1,155 @@
 
 # Matter Casting APIs
 
-  
+Matter Casting consists of three parts:
+
+* **The mobile app**: For most content providers, this would be your consumer-facing mobile app.  By making your moile app a Matter "Casting Client", you enable the user to discover casting targets, cast content, and control casting sessions.
```

Comment:
```suggestion
* **The mobile app**: For most content providers, this would be your consumer-facing mobile app.  By making your moile app a Matter "Casting Client", you enable the user to discover casting targets, cast content, and control casting sessions. The [example Matter tv-casting-app](https://github.com/project-chip/connectedhomeip/tree/master/examples/tv-casting-app) for Android / iOS and Linux builds on top of the Matter SDK to demonstrate how a TV Casting mobile app works.
```